pyaedt/desktop.py

```diff
@@ -589,16 +591,16 @@ def _exception(self, ex_value, tb_data):
         """
         try:
             oproject = self._main.oDesktop.GetActiveProject()
-            proj_name = oproject.GetName()
+            project_name = oproject.GetName()
             try:
-                des_name = oproject.GetActiveDesign().GetName()
-                if ";" in des_name:
-                    des_name = des_name.split(";")[1]
+                design_name = oproject.GetActiveDesign().GetName()
+                if ";" in design_name:
+                    design_name = design_name.split(";")[1]
             except:
-                des_name = ""
+                design_name = ""
         except:
-            proj_name = ""
-            des_name = ""
+            project_name = ""
```

Comment:
@maxcapodi78 This variable is assigned but never used. We could get rid of it.

Reply:
I agree